ui/src/templates/components/CommunityTemplateImportOverlay.tsx

```diff
@@ -62,6 +75,21 @@ class UnconnectedTemplateImportOverlay extends PureComponent<Props> {
     )
   }
 
+  private reviewTemplateResources = async (orgID, templateName) => {
+    const yamlLocation = `${getRawUrlFromGithub(
+      getGithubUrlFromTemplateName(templateName)
+    )}/${templateName}.yml`
+
+    try {
+      const summary = await reviewTemplate(orgID, yamlLocation)
+
+      this.props.setCommunityTemplateToInstall(summary)
+      return summary
```

Comment:
do we need this return if its only used in `componentDidMount`?

Reply:
@121watts  it's not currently used, but i always make a habit of returning promises from async functions to facilitate chaining. i thought it was spec for any function that resolves a promise to return it so it can be chained, but i can't find any references to that so i might be full of it...